packages/node_modules/@ciscospark/react-component-title-bar/src/styles.css

```diff
@@ -7,7 +7,6 @@
 
 .title {
   margin: 0;
-  padding-top: 10px;
```

Comment:
Does removing this still get the title to align with the top of the avatar? Was basing it off of this mock: https://app.zeplin.io/project.html#pid=5891363f1e71c1f114ec20a8&sid=5899302750e270570fe1d37f